components/feature/download/src/main/java/mozilla/components/feature/download/DownloadManagerFeature.kt

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.download
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.support.annotation.RequiresPermission
+import mozilla.components.browser.session.Download
+import mozilla.components.browser.session.Session
+import mozilla.components.browser.session.SessionManager
+
+/**
+ * Feature implementation for proving download functionality for the selected session.
+ */
+class DownloadManagerFeature(
```

Comment:
nit: I think for consistency it would be nice to use the same name as the module: `DownloadFeature`.

Originally I was thinking of `feature-downloads` (s) and `DownloadsFeature`. But the *s* is not that important :D

Reply:
^_^
I renamed the component to **feature-downloads** and `DownloadsFeature` 🕺 .